Components/WebLinkManager.php

```diff
@@ -48,7 +48,7 @@ public function link($uri, $rel, array $attributes = [])
             return $uri;
         }
 
-        if (strpos($uri, 'http') === 0) {
+        if (strpos($uri, 'http') === 0 && $this->front->Request()->getHttpHost() === parse_url($uri, PHP_URL_HOST)) {
```

Comment:
I would like more an option to the tags, to enable this feature or not instead doing it globally